src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -951,15 +943,170 @@ The description of the configuration options for the remaining ProActive compone
+ Running in interactive installation mode.
+You are going to set the passphrase for password encryption [y/n] y
+Enter Passphrase:
+Retype Passphrase:
+ Passphrase is set.
+Do you want to set ProActive admin password? [y/n] y
+Enter ProActive admin password:
+Retype ProActive admin password:
+ ProActive admin password is set.
+Do you want to set ProActive user password? [y/n] y
+Enter ProActive user password:
+Retype ProActive user password:
+ ProActive user password is set.
+Do you want to set the TLS certificate keystore password? [y/n] n
+Do you want to set LDAP configuration passwords? [y/n] n
+Do you want to set ProActive users passwords? [y/n] n
+Do you want to set ProActive external database passwords? [y/n] n
+...
+----
+
+If you want to start the ProActive server installation and configuration offline (without requiring internet access), you can include the `-o` option in the installation command to ensure it runs in offline mode:
+----
+$ sudo ./proactiveInstaller.sh -i -o
+----
+
+===== ProActive Server Installation with Passwords from a File
+
+This method allows for an automated installation of the ProActive server by using a file to provide the necessary clear-text passwords.
+
+It is ideal for users who want to simplify the installation process by pre-defining passwords in a configuration file. It is suitable for users who prefer to avoid manual input during installation, making it useful for automation.
+
+During this mode, you specify the installation mode using `-a` option (for automatic) and the path to a clear-text password file using the `-f` option (e.g., `-f ./password-template.properties`) that contains the clear-text passwords for various configurations, such as admin, user, LDAP, TLS, or database passwords.
+
+A pre-filled `password-template.properties` file is provided in the installation package, and you can use it to provide the clear-text passwords.
+If some advanced configurations are disabled in the `inputConfig.json` file (e.g., LDAP or database configuration are disabled), the relative passwords will be simply ignored.
+
+The installer will then use the provided passphrase in the password file to encrypt the passwords and insert them into the `inputConfig.json` file.
+
+Example:
 
 ----
-$ chmod +x proactiveInstaller.sh
-$ sudo ./proactiveInstaller.sh -p PASSPHRASE
+$ sudo ./proactiveInstaller.sh -a -f ./password-template.properties
+
+ Automatic installation mode is set.
+ Using file: ./password-template.properties for automatic installation.
+ Running in automatic installation mode.
+...
 ----
 
+If you want to start the ProActive server installation and configuration offline (without requiring internet access), you can include the `-o` option in the installation command to ensure it runs in offline mode:
+----
+$ sudo ./proactiveInstaller.sh -a -f ./password-template.properties -o
+----
+
+NOTE: After the installation is complete, it is recommended to delete the clear-text password file to enhance security.
+
```

Comment:
Maybe, delete this file automatically once the installation is complete?

Reply:
not really, it is needed for the upgrade for example

Reply:
I added this: NOTE: After the installation is complete, it is recommended to delete the clear-text password values to improve security